src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceReferenceContact.vue

```diff
@@ -75,6 +81,16 @@
             @update:model-value="certificateProvinceIdChanged"
             @click:clear="provinceClearClicked"
           ></v-autocomplete>
+          <v-autocomplete
+            v-if="wizardStore.wizardData.inviteType === PortalInviteType.WORK_EXPERIENCE"
+            :model-value="modelValue.certificateProvinceId"
+            label="Province/Territory Certified/Registered In"
+            variant="outlined"
+            color="primary"
+            :items="configStore?.provinceList.filter((province) => province.title !== ProvinceTerritoryType.OTHER)"
+            hide-details="auto"
+            @update:model-value="certificateProvinceIdChanged"
```

Comment:
this needs a a rule to be required for Work Experience References.

Reply:
Nice catch! added a "required" rule